crates/hotshot/src/tasks/mod.rs

```diff
@@ -619,6 +619,45 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES> + std::fmt::Debug, V: Version
     }
 }
 
+#[derive(Debug)]
+/// An `EventHandlerState` that modifies view number on `DacSend` to that of a a future view
```

Comment:
looks like i left a typo here
```suggestion
/// An `EventHandlerState` that modifies view number on the certificate of `DacSend` event to that of a future view
```